docs/.vitepress/config.mts

```diff
@@ -0,0 +1,103 @@
+import { defineConfig } from 'vitepress'
+
+// https://vitepress.dev/reference/site-config
+export default defineConfig({
+  ignoreDeadLinks: true,
+  base: '/zx/',
+  outDir: 'docs',
```

Comment:
This does mean, that `<root>/docs/docs` will be created, while _Upload artifact_ step is still referencing to `<root>/docs`

Reply:
Lets rename it to build?

Reply:
Good point, @antongolub. I renamed the artifact directory to `build`, as @antonmedv suggested, and amended workflow file and Vitepress configuration accordingly.